docs/scripts/generate_skills_docs.py

```
@@ -0,0 +1,150 @@
+class Skill:
+    """Represents a skill with its metadata."""
+
+    name: str
+    description: str
+    github_url: str
+    slug: str
+    category: str = "General"
+    emoji: str = "🤖"
+
+
+def _fetch(url: str, parse_json: bool = False):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_fetch`

(ANN202)